src/Config/StandardConfig.php

```diff
@@ -560,7 +562,7 @@ public function setCookieSecure($cookieSecure)
     /**
      * Get session.cookie_secure
      *
-     * @return bool
+     * @return bool|string
```

Comment:
Should we even support `string` here? Can't we cast in the return statement?

Reply:
The reason I didn't cast in the return statement is due to test failures that verify parity with `assertSame(ini_get('session.whatever'), $foo->getCookieSecure())`